rs/bitcoin/ckbtc/minter/BUILD.bazel

```diff
@@ -134,16 +134,16 @@ rust_ic_test(
     srcs = ["tests/tests.rs"],
     data = [
         ":ckbtc_minter_debug.wasm",
-        "//rs/bitcoin/ckbtc/kyt:kyt_canister.wasm",
-        "//rs/bitcoin/mock:bitcoin_canister_mock.wasm",
-        "//rs/rosetta-api/icrc1/ledger:ledger_canister.wasm",
+        "//rs/bitcoin/ckbtc/kyt:kyt_canister",
+        "//rs/bitcoin/mock:bitcoin_canister_mock",
+        "//rs/rosetta-api/icrc1/ledger:ledger_canister",
     ],
     env = {
         "CARGO_MANIFEST_DIR": "rs/bitcoin/ckbtc/minter",
         "IC_CKBTC_MINTER_WASM_PATH": "$(rootpath :ckbtc_minter_debug.wasm)",
-        "IC_ICRC1_LEDGER_WASM_PATH": "$(rootpath //rs/rosetta-api/icrc1/ledger:ledger_canister.wasm)",
```

Comment:
understanding question: Since the exact file is no longer specified (before: `ledger:ledger_canister.wasm`, after: `ledger:ledger_canister`), how do I know which file will be used for the test? There are other ones, such as `ledger:ledger_canister.wasm.gz`, which are available.

Reply:
it will actually use `ledger_canister.wasm` (which happens to be the same as `ledger_canister.wasm.gz`):

```bzl
    gzip_compress(
        name = name + ".wasm", # ledger_canister.wasm (compressed even though no .gz)
        srcs = [name + "_with_version.opt"],
        testonly = kwargs.get("testonly"),
    )


    native.alias(
        name = name, # alias to ledger_canister.wasm
        actual = name + ".wasm",
    )
```

[link](https://github.com/dfinity/ic/blob/1925b70649dd7f83f91aba61fb7ecbf891bc6a32/bazel/canisters.bzl#L61)

note that this PR is preparation work for https://github.com/dfinity/ic/pull/440 which will actually clean things up a bit

Reply:
Thanks for the explanation!